### PR TITLE
build & configuration of machine

### PR DESCRIPTION
**Pull request by benadida**
admin scripts in `tty2` are not fully working yet, some path issues to resolve with new setup, but everything else is in good shape.

### CODE REVIEW

File: config/xinitrc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+unclutter -idle 0.01 -root &
+
+if [ "${VX_MACHINE_TYPE}" = bmd ]; then
+    URL=http://localhost:3000/speech-loader.html
+else
+    URL=http://localhost:3000/
+fi
+
+exec sh <<SHELL
```
Comment (eventualbuddha) [inline config/xinitrc:15]: This works? I’d have expected `exec sh <<SHELL` for it to be terminated by `SHELL` later.

File: run-election-manager.sh [hunk 2]
```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
 
+# go to directory where this file is located
+cd "$(dirname "$0")"
```
Comment (eventualbuddha) [inline run-election-manager.sh:4]: This sometimes gives the wrong results, like when symlinks are involved. I’m guessing it’s fine for us?

File: setup-machine.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```
Comment (eventualbuddha) [inline setup-machine.sh:2]: `set -euo pipefail`? Safety First™

Comment (eventualbuddha) [reply setup-machine.sh:2]: Also on the other scripts like `setup-lockdown.sh`.

File: config/xinitrc (hunk 1 above)
Comment (benadida) [reply config/xinitrc:15]: I think something got mangled here. I'll check :)

File: run-election-manager.sh (hunk 2 above)
Comment (benadida) [reply run-election-manager.sh:4]: Yeah I think it's okay, but if you have a better suggestion, I'm not a shell master and welcome it.

File: config/xinitrc (hunk 1 above)
Comment (benadida) [reply config/xinitrc:15]: no, actually, not mangled, yes this works. I'm not sure I understand what you think it might do?

Comment (eventualbuddha) [reply config/xinitrc:15]: Oh. Weird. This was an issue with the Github iOS app. It only showed `exec sh <`.

File: run-election-manager.sh (hunk 2 above)
Comment (eventualbuddha) [reply run-election-manager.sh:4]: Here's a SO post on the subject: https://stackoverflow.com/a/246128/549363. The more correct version is… verbose. Let's just go with this for now.

**Pull request merged**